packages/user/.config/hakari.toml

```diff
@@ -0,0 +1,27 @@
+# This file contains settings for `cargo hakari`.
+# See https://docs.rs/cargo-hakari/latest/cargo_hakari/config for a full list of options.
+
+hakari-package = "user-workspace-hack"
+
+# Format version for hakari's output. Version 4 requires cargo-hakari 0.9.22 or above.
+dep-format-version = "4"
+
+# Setting workspace.resolver = "2" or higher in the root Cargo.toml is HIGHLY recommended.
+# Hakari works much better with the v2 resolver. (The v2 and v3 resolvers are identical from
+# hakari's perspective, so you're welcome to set either.)
+#
+# For more about the new feature resolver, see:
+# https://blog.rust-lang.org/2021/03/25/Rust-1.51.0.html#cargos-new-feature-resolver
+resolver = "2"
+
+# Add triples corresponding to platforms commonly used by developers here.
+# https://doc.rust-lang.org/rustc/platform-support.html
+platforms = [
+    "x86_64-unknown-linux-gnu",  # Linux x86_64 (primary dev platform)
+    "aarch64-unknown-linux-gnu", # Linux ARM64 (container platform, ARM servers)
+    "wasm32-wasip1",             # CRITICAL: psibase services compile to WebAssembly
```

Comment:
Is `wasm32-wasip1` needed here?
It's a build target, but this looks like it's supposed to be a list of host platforms.